app/javascript/src/components/Invoices/Invoice/index.tsx

```diff
@@ -1,16 +1,46 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { setAuthHeaders, registerIntercepts } from "apis/axios";
+import invoicesApi from "apis/invoices";
 import Header from "./Header";
 import InvoiceDetails from "./InvoiceDetails";
+import { ApiStatus as InvoiceStatus } from "../../../constants";
 
-const Invoice = ({ invoice }) => (
-  <>
-    <div className="max-w-6xl mx-auto px-2 md:px-11 font-manrope">
-      <Header invoice={invoice} />
-      <div className="bg-miru-gray-100 mt-5 mb-10 p-0 m-0 w-full">
-        <InvoiceDetails invoice={invoice}/>
-      </div>
-    </div>
-  </>
-);
+const Invoice = () => {
+  const params = useParams();
+
+  const [status, setStatus] = React.useState<InvoiceStatus>(
+    InvoiceStatus.IDLE
+  );
+  const [invoice, setInvoice] = useState<any>();
```

Comment:
Just one tiny change, the initial value should be `null`.
```suggestion
  const [invoice, setInvoice] = useState<any>(null);
```

Reply:
Fixed.